packages/esm-patient-programs-app/src/programs/programs-form.component.tsx

```diff
@@ -54,13 +54,19 @@ type ViewState = IdleState | ResolvedState | SubmittingState;
 const ProgramsForm: React.FC<ProgramsFormProps> = ({ patientUuid, closeWorkspace }) => {
   const { t } = useTranslation();
   const session = useSessionUser();
+  const layout = useLayoutType();
   const [availableLocations, setAvailableLocations] = React.useState(null);
   const [completionDate, setCompletionDate] = React.useState(null);
   const [enrollmentDate, setEnrollmentDate] = React.useState(new Date());
   const [userLocation, setUserLocation] = React.useState('');
   const [viewState, setViewState] = React.useState<ViewState>({
     type: StateTypes.IDLE,
   });
+  const [lightMode, setLightMode] = useState<boolean>();
```

Comment:
I see a pattern emerging here.

Reply:
Moved this logic to `context-workspace` because most of the forms require this prop